packages/ERTP/src/issuer.js

```diff
@@ -200,6 +214,7 @@ function produceIssuer(allegedName, mathHelpersName = 'nat') {
     // checking is delegated to the `reallocate` function.
     combine: (fromPaymentsPArray, optTotalAmount = undefined) => {
       return Promise.all(fromPaymentsPArray).then(fromPaymentsArray => {
+        fromPaymentsArray.every(p => assertKnownPayment(p));
```

Comment:
I think this could just be:
```suggestion
        fromPaymentsArray.every(assertKnownPayment);
```